quarkchain/evm/messages.py

```diff
@@ -578,6 +583,15 @@ def _apply_msg(ext, msg, code):
     if res == 0:
         log_msg.debug("REVERTING")
         ext.revert(snapshot)
+    if (
```

Comment:
I think we should reuse the previous code as

if res == 0 and code != b"" and msg.transfer_token_id != ext.default_state_token and not ext.token_id_quieried:
  res = 1

Reply:
Also may skip checking if transfer value is zero.